tests/test_file_utils.py

```diff
@@ -85,3 +109,26 @@ def test_lfs_object(self):
         filepath = get_from_cache(url, force_download=True)
         metadata = filename_to_url(filepath)
         self.assertEqual(metadata, (url, f'"{PINNED_SHA256}"'))
+
+
+class ContextManagerTests(unittest.TestCase):
+    @unittest.mock.patch("sys.stdout", new_callable=io.StringIO)
+    def test_no_context(self, mock_stdout):
+        with ContextManagers([]):
+            print("Transformers are awesome!")
+        # The print statement adds a new line at the end of the output
+        self.assertEqual(mock_stdout.getvalue(), "Transformers are awesome!\n")
```

Comment:
btw we also have a context manager for catching std streams.
https://huggingface.co/transformers/master/testing.html#testing-the-stdout-stderr-output
scroll down to CaptureStd

either way works, just sharing we already have a built-in for that functionality.